src/core/blockchain/types/token.ts

```diff
@@ -37,5 +37,7 @@ export enum PosBasicActionType {
     WITHDRAW = 'WITHDRAW',
     REINVEST = 'REINVEST',
     LOCK = 'LOCK',
-    CREATE_ACCOUNT = 'CREATE_ACCOUNT'
+    CREATE_ACCOUNT = 'CREATE_ACCOUNT',
```

Comment:
create_account is a PosAction? its not something that happens to have accounts for transfer?